AdobeIms/Model/UserProfileRepository.php

```diff
@@ -106,7 +106,7 @@ public function getByUserId(int $userId): UserProfileInterface
         $entity = $this->entityFactory->create();
         $this->resource->load($entity, $userId, self::ADMIN_USER_ID);
         if (!$entity->getId()) {
-            $message = __('User profile with user id %id not found.', ['id' => $userId]);
+            $message = __('User profile with user id %id not found.', [self::ID => $userId]);
```

Comment:
This is a mistake, array key is applied to '%id' in there string nearby, while ID constant is a column to load data.